chef/cookbooks/ceph/templates/default/ceph.conf.erb

```diff
@@ -141,3 +141,10 @@
   <% end -%>
 <% end -%>
 <% end -%>
+
+<% if (! node['ceph']['config_sections'].nil?) -%>
```

Comment:
after talking this @toabctl, we agree the this `.nil?` check is a safety net in case of a random `node['ceph'].delete("config_sections")`